pkgs/development/python-modules/apt-repo/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  unstableGitUpdater,
+
+  buildPythonPackage,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "apt-repo";
+  version = "0-unstable-2023-09-27";
```

```suggestion
  version = "0.5-unstable-2023-09-27";
```
Looking at setup.py and PyPI, it was released as version 0.5.
https://github.com/brennerm/python-apt-repo/blob/0287c59317f9ec8e8edbf7c228665a7010f758e7/setup.py#L10
https://pypi.org/project/apt-repo/